lib/Parameters/BGV/Params.cpp

```diff
@@ -0,0 +1,73 @@
+#include "lib/Parameters/BGV/Params.h"
+
+#include <cassert>
+
+namespace mlir {
+namespace heir {
+namespace bgv {
+
+struct RLWEParam {
+  int ringDim;
+  int maxQ;
+};
+
+// uniform tenary
```

Comment:
```suggestion
// parameters for uniform ternary secret distribution
```

Reply:
I don't recall if BGV has a sparsity condition for its secret distribution, but if so please add the hamming weight. Or better yet, if you can cite an external doc that would be helpful.

Reply:
In the homormorphic encryption standard there is no sparse setting. Added link to HES and Openfhe.